### PR TITLE
treewide: per RFC45, remove more unquoted URLs

### PR DESCRIPTION
**Pull request by prusnak**
###### Motivation for this change

This PR deals with the rest of the unquoted URLs in the tree.

I also included occurrences in comments and documentation just to be consistent.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by 7c6f434c — APPROVED** [review]

**Comment by 7c6f434c** [comment]
@ofborg test nixos-functions.nixos-test

(I wonder if that works)

File: pkgs/development/haskell-modules/configuration-tensorflow.nix [hunk 1]
```diff
@@ -55,12 +55,12 @@ in
...
     # Adapt to the Semigroup–Monoid Proposal (enables building on GHC >= 8.4)
     (pkgs.fetchpatch {
-        url = https://github.com/haskell-suite/haskell-src-exts/commit/258e072fe9e37f94360b7488b58ea2832843bbb8.patch;
+        url = "https://github.com/haskell-suite/haskell-src-exts/commit/258e072fe9e37f94360b7488b58ea2832843bbb8.patch";
         sha256 = "0ja6ai41v9plinlhjwja282m6ahn6mw4xi79np0jxqk83cg0z1ff";
     })
     # Adapt to MonadFail proposal (enables building on GHC >= 8.8)
     (pkgs.fetchpatch {
-        url = https://gist.githubusercontent.com/mikesperber/0f2addaf3fbe97ffb4a5234d8711ba41/raw/e09e20998283c7195e82d546ba9266d290eb000d/gistfile1.txt;
+        url = "https://gist.githubusercontent.com/mikesperber/0f2addaf3fbe97ffb4a5234d8711ba41/raw/e09e20998283c7195e82d546ba9266d290eb000d/gistfile1.txt";
```
Comment (cdepillabout) [inline pkgs/development/haskell-modules/configuration-tensorflow.nix:63]: Oops, sorry, I let these through just recently.

cc @mikesperber as well

**Comment by prusnak** [comment]
Thanks for the quick merge. AFAIK there are no unquoted URLs in the whole nixpkgs tree now.

**Comment by mikesperber** [comment]
@cdepillabout Thanks!

**Pull request merged**